paymentsheet/src/main/java/com/stripe/android/paymentsheet/analytics/PaymentSheetEvent.kt

```diff
@@ -301,6 +327,7 @@ internal sealed class PaymentSheetEvent : AnalyticsEvent {
         const val FIELD_COLLECT_PHONE = "phone"
         const val FIELD_COLLECT_ADDRESS = "address"
         const val FIELD_DURATION = "duration"
+        const val FIELD_SELECTED_LPM = "selected_lpm"
```

Comment:
What do values is android sending for `selected_lpm`? Can I see some examples?

Reply:
The value is coming from `SupportedPaymentMethod.code` (see [here](https://github.com/stripe/stripe-android/blob/c91c1d169987dd608d5fe53227838e71b59471e5/payments-ui-core/src/main/java/com/stripe/android/ui/core/forms/resources/LpmRepository.kt#L239)).